library/src/main/java/com/cube/storm/ui/activity/VideoPlayerActivity.java

```diff
@@ -94,6 +95,15 @@ public void onCreate(Bundle savedInstanceState)
 		progressBar = findViewById(R.id.progress);
 		playerView.requestFocus();
 
+		// ARCFA-239 Don't hide video controls when screen reader is on
```

Comment:
Well done for putting ticket refs in throughout this PR. All nicely structured and easy to follow